### ISSUE TITLE
[DOC] Missing documentation on how to connect multiple OCP clusters to an F5 device in cluster mode

### ISSUE THREAD

**Issue by rheinzma** [F5Networks/k8s-bigip-ctlr#1289]
#### Description

Currently there is no documentation on how to connect multiple OCP clusters to a single F5 device properly in cluster mode if they have the same machineCIDR as well as the same podNetwork range. This is a common setup for OpenShift 4 IPI installations where the cluster is setup in a very opinionated way (e.g. on OpenStack). 

There is some basic hints to route domains, however setup of route domains and partitions for the different clusters is not well documented as of today. 

Also as of https://github.com/F5Networks/k8s-bigip-ctlr/issues/1287 and https://github.com/F5Networks/k8s-bigip-ctlr/issues/1288 there seems to be issues with the usage of the AS3 mode with route domains, which is somehow the foundation for the setup I assume.

#### Kubernetes Version

OpenShift 4.4.3 on OpenStack IPI

#### Controller Version

1.14

#### BIG-IP Version

BIG-IP 13.1.3.3 Build 0.0.6 Point Release 3

**Comment by rheinzma** [comment]
It seems the release notes (See [Here](https://github.com/F5Networks/k8s-bigip-ctlr/blob/master/docs/RELEASE-NOTES.rst)) states: 

```
The BIG-IP Controller does not handle non-zero route domains. All managed partitions should use the default route domain (0).
```

This somehow conflicts with the statement on the website for OpenShift (See [here](https://clouddocs.f5.com/containers/v2/openshift/kctlr-openshift-app-install.html): 

```
[Optional] The Controller can declare the IP addresses it configures on the BIG-IP with a Route Domain identifier. You may want to use route domains if you have many applications using the same IP address space that need isolation from one another. After you create the partition on your BIG-IP system, you can 1) create a route domain and 2) assign the route domain as the partition’s default. See create and set a non-zero default Route Domain for a partition for setup instructions.
```

**Comment by mdditt2000** [comment]
#1287 and #1288 are resolved in CIS 2.1 Closing this issue out.

**Comment by timbrd** [comment]
@rheinzma Did you find a way to connect multiple OCP clusters to a single BigIP Cluster using the same or overlapping IP ranges for machine network and pod network?

I'm still a bit puzzled. I tend to assigning network ranges which don't overlap, but since I have many OpenShift clusters, I have to shrink the ip ranges.
